com.pypyrev.linuxdcpp.json

```diff
@@ -0,0 +1,74 @@
+{
+	"app-id": "com.pypyrev.linuxdcpp",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "18.08",
```

Comment:
this runtime is deprecated. Please use 20.08

Reply:
Yes, I know that it is deprecated. But I wanted compatibility with 32-bit systems, and this seems to be possible only with old runtime.

So, should I change it to 20.08 anyway?

Reply:
yes.

i386 is not supported anymore. Nor is arm.